Another projekts/calculator/main_calculator_1.py

```diff
@@ -0,0 +1,15 @@
+from functions import mathematical_operation
```

Comment:
That's a great solution for your studying in my opinion but do not overdo this in the other projects. This is can be a false friend when you have a really simple program and not planning to use this module in other projects. In this case, such complications can make worse code clarity.

Reply:
I will take that into consideration